blocks/model.py

```diff
@@ -50,18 +50,18 @@ class AbstractModel(object):
 
     """
     @abstractmethod
-    def get_params(self):
+    def get_parameter_dict(self):
         """Return the model parameters.
 
         Returns
         -------
-        params : OrderedDict
+        parameters : OrderedDict
             Dictionary of (name, parameter) pairs.
 
         """
         pass
```

Comment:
No. There is no self.parameters in AbstractModel. It's up to AbstractModel childs to define how they build parameter_dict. [Model](https://github.com/mila-udem/blocks/pull/725/files#diff-ff403bd4d4da2f194bd025e38a075649R122) class inherits from ComputationGraph, that's where self.parameters comes from.